tests/manual/performance/editorinit.md

```diff
@@ -0,0 +1,8 @@
+# Performance: editor initialization
+
+1. You might want to enable devtools profiling.
```

Comment:
This steps should describe to other guys how to measure performance (in short). So I'd not use `might` or `optional` here - just the steps required to perform the check.